README.md

```diff
@@ -37,33 +37,77 @@ These steps can be followed to use with a virtual environment and `pip`:
 
 The SDK is now ready for use.
 
-### Starting a Workflow
+### Implementing a Workflow
 
-Create the following script at `start_workflow.py`:
+Create the following script at `run_worker.py`:
 
 ```python
 import asyncio
+from datetime import datetime, timedelta
+from temporalio import workflow, activity
 from temporalio.client import Client
+from temporalio.worker import Worker
 
-async def main():
-  # Create client connected to server at the given address
-  client = await Client.connect("http://localhost:7233")
+@activity.defn
+async def say_hello(name: str) -> str:
+    return f"Hello, {name}!"
+
+@workflow.defn
+class SayHello:
+    @workflow.run
+    async def run(self, name: str) -> str:
+        return await workflow.execute_activity(
```

Comment:
While you can, I am intentionally encouraging people do it this way so every workflow thing starts with `workflow`. So you know that everything `workflow.` is for workflows and all functions therein can only be called in a workflow context. In Python, it's common to somewhat qualify otherwise generic method names instead of forcing people to re-read the import set constantly.

It'd be rare to see something like a naked `info()` in Python code, because it'd force readers to refer to the import to understand what that means. Same for Go. In JS/TS it's much more common to either over-qualify the name itself or force people to read the imports for context. Same reason you see something like `asyncio.sleep()` instead of just `sleep()` in Python code.

Of course it's user preference, but I think it's important for the examples. Also, there are some ambiguities between this and the `activity` module. E.g. `@defn`, `info()`, etc.